sdk/trace/provider.go

```diff
@@ -93,16 +95,28 @@ func NewProvider(opts ...ProviderOption) (*Provider, error) {
 
 // Tracer with the given name. If a tracer for the given name does not exist,
 // it is created first. If the name is empty, DefaultTracerName is used.
-func (p *Provider) Tracer(name string) apitrace.Tracer {
+func (p *Provider) Tracer(name string, opts ...apitrace.TracerOption) apitrace.Tracer {
+	c := new(apitrace.TracerConfig)
+	for _, o := range opts {
+		o(c)
+	}
 	p.mu.Lock()
 	defer p.mu.Unlock()
 	if name == "" {
 		name = defaultTracerName
+		c.InstrumentationVersion = sdk.Version()
```

Comment:
I'm not sure why this makes sense. Won't the sdk.Version be a different resource value?

Reply:
I'm not sure I follow how this becomes a resource value, but I don't think it is too important.

Since the specification doesn't make a statement here, I'm just going to remove this to get it rolling.

Reply:
Removed.

Reply:
The specification defines `telemetry.sdk.name`, `telemetry.sdk.language`, and `telemetry.sdk.version`.  The specification says that the instrumentation version is optional, which leads me to believe that we can leave it empty and not create a tag when the value is empty.

Reply:
Ah, that makes sense. Sounds like we made the correct decision. :smile: